app/styles/base/_slider.less

```diff
@@ -4,9 +4,11 @@
 
     .oss-slider__tooltip {
       text-align: center;
+      white-space: nowrap;
       left: var(--tooltip-left-position);
       top: 30px;
       transition: opacity 300ms;
+      z-index: 1000;
```

Comment:
I think this was one of the attempts to have it display over the tether-element in the crm context.
Don't think it's required anymore ?

Reply:
Cleaned up